changelog/2248.fixed.md

```diff
@@ -0,0 +1,2 @@
+Fixed solaris build globally, also disabling a subset of `InterfaceFlags` values
```

Comment:
You're mixing past and present tense in the same sentence here.  And what does the second part mean?  Without reading the code, I can't understand why you disabled a subset of `InterfaceFlags`.